cf-agent/cf-agent.c

```diff
@@ -179,6 +179,13 @@ static const char *const CF_AGENT_MANPAGE_LONG_DESCRIPTION =
         "For each bundle, cf-agent groups promise statements according to their type. Promise types are then evaluated in a preset "
         "order to ensure fast system convergence to policy.\n";
 
+static const Component COMPONENT =
+{
+    .name = "cf-agent",
+    .website = "https://cfengine.com",
+    .copyright = "2023 Northern.tech AS"
```

Comment:
Website and copyright should be macros defined in either `cf3.defs.h` or in `config.h` (by means of autotools). We don't want to update them in multiple places.